koku/masu/database/presto_sql/reporting_ocpazurecostlineitem_daily_summary.sql

```diff
@@ -51,71 +145,104 @@ WITH cte_ocp_on_azure_resource_id_joined AS (
+    sum(ocp.pod_request_cpu_core_hours) as pod_request_cpu_core_hours,
+    sum(ocp.pod_limit_cpu_core_hours) as pod_limit_cpu_core_hours,
+    sum(ocp.pod_usage_memory_gigabyte_hours) as pod_usage_memory_gigabyte_hours,
+    sum(ocp.pod_request_memory_gigabyte_hours) as pod_request_memory_gigabyte_hours,
+    max(ocp.cluster_capacity_cpu_core_hours) as cluster_capacity_cpu_core_hours,
+    max(ocp.cluster_capacity_memory_gigabyte_hours) as cluster_capacity_memory_gigabyte_hours,
+    max(ocp.pod_labels) as pod_labels,
+    NULL as volume_labels,
+    max(azure.tags) as tags,
+    row_number() OVER (partition by azure.uuid) as project_rank,
+    1 as data_source_rank,
+    max(azure.resource_id_matched) as resource_id_matched
     FROM hive.{{schema | sqlsafe}}.azure_openshift_daily as azure
-    JOIN postgres.{{schema | sqlsafe}}.reporting_ocpusagelineitem_daily_summary as ocp
-        ON coalesce(azure.date, azure.usagedatetime) = ocp.usage_start
-            AND (
+    JOIN hive.{{schema | sqlsafe}}.reporting_ocpusagelineitem_daily_summary as ocp
+        on coalesce(azure.date, azure.usagedatetime) = ocp.usage_start
+        -- I kept this AND condition the same, I notice Andrew removed the data source check on the ocp on aws
+        AND (
                 (split_part(coalesce(azure.resourceid, azure.instanceid), '/', 9) = ocp.node AND ocp.data_source = 'Pod')
                     OR (split_part(coalesce(azure.resourceid, azure.instanceid), '/', 9) = ocp.persistentvolume AND ocp.data_source = 'Storage')
             )
     WHERE azure.source = '{{azure_source_uuid | sqlsafe}}'
-        AND azure.year = '{{year | sqlsafe}}'
-        AND azure.month = '{{month | sqlsafe}}'
+        AND azure.year = {{year}}
+        AND azure.month = {{month}}
         AND coalesce(azure.date, azure.usagedatetime) >= TIMESTAMP '{{start_date | sqlsafe}}'
         AND coalesce(azure.date, azure.usagedatetime) < date_add('day', 1, TIMESTAMP '{{end_date | sqlsafe}}')
-        AND ocp.report_period_id = {{report_period_id | sqlsafe}}
-        AND ocp.usage_start >= date('{{start_date | sqlsafe}}')
-        AND ocp.usage_start <= date('{{end_date | sqlsafe}}')
-    GROUP BY azure.uuid, ocp.namespace, ocp.data_source
-),
-cte_ocp_on_azure_tag_joined AS (
-    SELECT azure.uuid as azure_id,
-        max(split_part(coalesce(resourceid, instanceid), '/', 9)) as resource_id,
-        max(coalesce(date, usagedatetime)) as usage_start,
-        max(coalesce(date, usagedatetime)) as usage_end,
-        max(json_extract_scalar(json_parse(azure.additionalinfo), '$.ServiceType')) as instance_type,
-        max(coalesce(subscriptionid, subscriptionguid)) as subscription_guid,
-        max(azure.resourcelocation) as resource_location,
+        AND (azure.resourceid IS NOT NULL AND azure.resourceid != '')
+        AND ocp.source = '{{ocp_source_uuid | sqlsafe}}'
+        AND ocp.year = {{year}}
+        AND ocp.month = {{month}}
```

Comment:
We may run into problems here with single digit months like we did with AWS, we had to pad the numbers to make sure the dates would line up, [PR1](https://github.com/project-koku/koku/pull/3362) --- [PR2](https://github.com/project-koku/koku/pull/3380)